tests/ui/use_self_trait.fixed

```diff
@@ -0,0 +1,118 @@
+// run-rustfix
+
+#![warn(clippy::use_self)]
+#![allow(dead_code)]
+#![allow(clippy::should_implement_trait)]
+
+#[allow(clippy::boxed_local)]
+mod traits {
```

Comment:
This `mod` isn't required anymore and the allow could be an inner attribute.

Reply:
good point, fixed.